src/cli/commands/create.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Create subnet cli command handler.
+
+use async_trait::async_trait;
+use clap::Args;
+use fvm_shared::clock::ChainEpoch;
+use std::fmt::Debug;
+
+use crate::cli::CommandLineHandler;
+use crate::config::json_rpc_methods;
+use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl};
+use crate::server::{CreateSubnetParams, CreateSubnetResponse};
+
+/// The command to create a new subnet actor.
+pub(crate) struct CreateSubnet;
+
+#[async_trait]
+impl CommandLineHandler for CreateSubnet {
+    type Arguments = CreateSubnetArgs;
+
+    async fn handle(arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("launching json rpc server with args: {:?}", arguments);
+
+        let params = CreateSubnetParams {
+            parent: arguments.parent.clone(),
+            name: arguments.name.clone(),
+            min_validator_stake: arguments.min_validator_stake,
+            min_validators: arguments.min_validators,
+            finality_threshold: arguments.finality_threshold,
+            check_period: arguments.check_period,
+        };
+
+        let url = arguments.ipc_agent_url.parse()?;
+        let json_rpc_client = JsonRpcClientImpl::new(url, None);
```

Comment:
We may need this for all other functions and we may be able to take it out to a common func.

Reply:
If I understand correctly, in this CLI, it's only a json rpc client initialization. The `JsonRpcClientImpl::new` is already the most simple form. Or are you referring to the initialization of LotusJsonRpcClient?